R-package/tests/testthat/test_Predictor.R

```diff
@@ -240,9 +240,7 @@ test_that("predict() params should override keyword argument for raw-score predi
         , nm = rawscore_alias
       )
     )
-    preds_raw_s3_param <- predict(bst, X, params = params)
     preds_raw_r6_param <- bst$predict(X, params = params)
-    expect_equal(preds_raw_s3_keyword, preds_raw_s3_param)
```

Comment:
Sorry, I just noticed this. @david-cortes , why are you proposing removing support for passing something like `"predict_raw_score"` through params?

That effectively reverts part of #5122, which was a fix for an issue you opened (#4670).

I feel that `{lightgbm}` should continue to support changing the type of prediction via `params` entries, and that `params` entries should take precedence over the `type` keyword argument in `predict.lgb.Booster()`, unless you can provide a compelling version to remove that support as part of this PR.

Reply:
Restored them.